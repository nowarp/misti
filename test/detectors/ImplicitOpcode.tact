message M1 { s: Slice as remaining; } // Bad
message(0x7362d09c) M2 { s: Slice as remaining; } // OK
