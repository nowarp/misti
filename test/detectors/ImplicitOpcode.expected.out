[INFO] ImplicitOpcode: Message `M1` has an implicit opcode
test/detectors/ImplicitOpcode.tact:1:1:
> 1 | message M1 { s: Slice as remaining; } // Bad
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  2 | message(0x7362d09c) M2 { s: Slice as remaining; } // OK
Help: Prefer explicitly defined message opcodes
See: https://nowarp.io/tools/misti/docs/detectors/ImplicitOpcode